src/OrganisationUnitTree.js

```diff
@@ -0,0 +1,235 @@
+/* eslint-disable */
```

Comment:
Quite a few errors after removing this line. Please check

Reply:
Yeah the reason why this is in there is because storybook won't compile for some reason until all lint issues have been fixed. Which means that developing like that is quite sh**.. So I added this comment until the PR could be approved, then I just need to run `yarn format:js`, which will fix the issues automatically